modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -1542,6 +1542,10 @@ public OperationsMap postProcessOperationsWithModels(OperationsMap objs, List<Mo
         OperationMap operations = objs.getOperations();
         List<CodegenOperation> operationList = operations.getOperation();
         for (CodegenOperation op : operationList) {
+            // check if the operation has form parameters
+            if (op.getHasFormParams()) {
```

Comment:
Maybe this condition could be extended to
```java
if (op.getHasFormParams() && !additionalProperties.containsKey("hasFormParamsInSpec")) {
```
? I am not very familiar with the code, but I think the `additionalProperties` can be set by the user, right? This way, the user could force to enable or disable the formdata-support.

This is not very important to me, but it may be important for others, and it seems really easy to do.

Reply:
Otherwise, these changes look perfectly fine to me 👌

Reply:
@ChristianCiach thanks for the review and the feedback. At this stage, I do not want to let users set this value to whatever they want.

If someone can later present a valid use case, happy to reconsider.